install_config/http_proxies.adoc

```diff
@@ -65,8 +80,8 @@ networkConfig:
   serviceNetworkCIDR: 172.30.0.0/16
 ----
 
-{product-title} does not accept `*` as a wildcard attached to a domain suffix. For
-example, this works:
+{product-title} does not accept `*` as a wildcard attached to a domain suffix.
+For example, this works:
```

Comment:
For me, "this works" doesn't feel right. 
"For example, the following configuration is accepted:"
"However, this configuration would net be accepted"
Or not...